backend/app/services/channels/telegram/keyboard.py

```
@@ -0,0 +1,219 @@
+            current_model_name: Currently selected model name (to mark with ✓)
+
+        Returns:
+            InlineKeyboardMarkup with model selection buttons
+        """
+        keyboard = []
+        row = []
+
+        for idx, model in enumerate(models, start=1):
+            model_name = model.get("name", "")
+            display_name = model.get("displayName") or model_name
+            provider = model.get("provider", "")
```

Ruff (0.14.14)

[error] Local variable `provider` is assigned to but never used

Remove assignment to unused variable `provider`

(F841)